datasync/chaindatafetcher/kas/repository.go

```diff
@@ -0,0 +1,42 @@
+package kas
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jinzhu/gorm"
+	"github.com/klaytn/klaytn/log"
+)
+
+const (
+	maxTransactionCount         = int64(1000000)
+	maxTransactionLogCount      = int64(100000)
+	maxInternalTransactionCount = int64(10000)
+
+	maxPlaceholders = 65535
+
+	placeholdersPerTxItem = 13
+)
+
+var logger = log.NewModuleLogger(log.ChainDataFetcher)
+
+type repository struct {
+	db *gorm.DB
+}
+
+func getEndpoint(user, password, host, port, name string) string {
+	return user + ":" + password + "@tcp(" + host + ":" + port + ")/" + name + "?parseTime=True&charset=utf8mb4"
+}
+
+func NewRepository(user, password, host, port, name string) *repository {
+	endpoint := getEndpoint(user, password, host, port, name)
+	db, err := gorm.Open("mysql", endpoint)
+	if err != nil {
+		logger.Crit("Connecting to DB is failed", "endpoint", endpoint, "err", err)
```

Comment:
How about try again with error log?

Reply:
Reconnect logic is added on f1f349c67257ed5a4fb0833cfcf62805d65744a2